ui/js/graphs.js

```diff
@@ -688,6 +688,9 @@ perf.controller('GraphsCtrl', [
                 timeRange: function() {
                   return $scope.myTimerange.value;
                 },
+                testCounter: function() {
```

Comment:
I don't think this needs to be a function, you can just set it to the value of $scope.seriesList.length.

Let's change the name from `testCounter` to `numTestsDisplayed`. More descriptive.